lib/box/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+var box = angular.module('guBox', []);
+
+box.directive('guBox', function boxDirective(){
+  return {
+    restrict: 'AE',
```

Comment:
This is where my angular falls off. Do you know how to identify if a directive is an element or an attribute?